### PR TITLE
[JA] Updating topics with the latest changes - 20 Oct - vnext

### PR DESCRIPTION
**Pull request by randriova**

### CODE REVIEW

**Comment by randriova** [comment]
@ChronosSF Can you wait for this PR to be merged before doing another mass merge to master, so that we can also release the JP changes? Thanks

**Review by DenitsaHristova — APPROVED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**
